.buildkite/x-pack/pipeline.xpack.auditbeat.yml

```diff
@@ -406,3 +410,43 @@ steps:
         notify:
           - github_commit_status:
               context: "x-pack/auditbeat: Packaging Linux arm64"
+
+      - label: ":ubuntu: x-pack/auditbeat: Packaging Linux amd64 FIPS"
+        key: "packaging-linux-fips"
+        env:
+          PLATFORMS: "${PLATFORMS_AMD64_FIPS}"
+          FIPS: "true"
+        command: |
+          .buildkite/scripts/packaging/packaging.sh x-pack/auditbeat
+        retry:
+          automatic:
+            - limit: 1
+        timeout_in_minutes: 20
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_HI_PERF_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        notify:
+          - github_commit_status:
+              context: "x-pack/auditbeat: Packaging Linux amd64 FIPS"
+
+      - label: ":ubuntu: x-pack/auditbeat: Packaging Linux arm64 FIPS"
+        key: "packaging-arm-fips"
+        env:
+          PLATFORMS: "${PLATFORMS_ARM64_FIPS}"
+          FIPS: "true"
```

Comment:
```suggestion
          FIPS: true
          
```

Reply:
Done in f35490f4f2b69447ed85f6716817e581bee82e95